src/tests/FunctionalTests/iOS/Simulator/XmlSerializer_Deserialize/Program.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+public static class Program
+{
+    [DllImport("__Internal")]
+    public static extern void mono_ios_set_summary (string value);
+
+    public static async Task<int> Main(string[] args)
+    {
+        mono_ios_set_summary($"Starting functional test");
```

Comment:
Is it really necessary to use these internal calls in tests?

Reply:
Hmmm, this internal call is actually in all existing tests under `src/tests/FunctionalTests`. Do we need it in all of them?